packages-native/progress-bar/src/__tests__/ProgressBar.spec.tsx

```diff
@@ -17,18 +17,18 @@ describe("ProgressBar", () => {
     it("renders no progress with undefined values", () => {
         const component = render(<ProgressBar {...createProps()} />);
         expect(component.getByType(Bar).props.progress).toBe(0);
-        expect(component.queryByType(Text)).toBeNull();
+        expect(component.queryByType(Text)).toBeDefined();
     });
 
-    it("renders no progress and an error when minimum equals maxiumum", () => {
+    it("renders no progress and an error when minimum equals maximum", () => {
         const component = render(<ProgressBar {...createProps(50, 50, 50)} />);
         expect(component.getByType(Bar).props.progress).toBe(0);
         expect(component.getByType(Text).props.children).toBe(
             "The minimum value must be equal or less than the maximum value."
         );
     });
 
-    it("renders no progress and an error when the value is less than the minium", () => {
+    it("renders no progress and an error when the value is less than the minimum", () => {
         const component = render(<ProgressBar {...createProps(-50, 0, 100)} />);
         expect(component.getByType(Bar).props.progress).toBe(0);
         expect(component.getByType(Text).props.children).toBe(
```

Comment:
We shouldn't expect any errors anymore ?

Reply:
We should.. even setting we should give visual feedback that the data is incorrect